src/Feed/KobaFeedType.php

```diff
@@ -80,7 +80,7 @@ public function getData(Feed $feed): array
                     if (!is_string($title)) {
                         $this->logger->error('KobaFeedType: event_name is not string.');
 
-                        throw new MissingFeedConfigurationException('Koba event_name is not string');
+                        throw new \Exception('Koba event_name is not string');
```

Comment:
Throwing an [`InvalidArgumentException`](https://www.php.net/manual/en/class.invalidargumentexception.php) may be more appropriate.